### DISCUSSION TITLE
Build gnoland to Trustwallet,

### DISCUSSION THREAD

**Discussion by iandareen** [gnolang/gno#1731] [Ideas]
Add gnoland to trust wallet more user more holder.

**Comment by thehowl** [comment]
Hey @iandareen! Currently we're still very actively building Gno.land, so getting added to existing wallets is not our top priority as we still need to be able to quickly iterate :) In the meantime, you may want to try out [adena](https://www.adena.app/). Have fun!

**Reply by iandareen** [reply]
Hoping soon ☺️☺️ have a nice day. Goodluck to your project. I'm waiting hehe.
